README.md

```diff
@@ -17,94 +17,6 @@ limitations under the License.
 [![License](https://img.shields.io/badge/License-Apache_2.0-lightgrey.svg)](https://opensource.org/licenses/Apache-2.0)
 
 # Triton Model Analyzer
-
-**LATEST RELEASE: You are currently on the main branch which tracks
-under-development progress towards the next release. The latest
-release of the Triton Model Analyzer is 1.28.0 and is available on
-branch
-[r23.05](https://github.com/triton-inference-server/model_analyzer/tree/r23.05).**
-
-Triton Model Analyzer is a CLI tool which can help you find a more optimal configuration, on a given piece of hardware, for single, multiple, ensemble, or BLS models running on a [Triton Inference Server](https://github.com/triton-inference-server/server/). Model Analyzer will also generate reports to help you better understand the trade-offs of the different configurations along with their compute and memory requirements.
```

Comment:
Was everything here supposed to be removed? I thought just the *LATEST RELEASE* was supposed to be removed?

Reply:
It will be updated on release date.